deeptools/computeMatrix.py

```diff
@@ -0,0 +1,418 @@
+                          'should be included or not. Default is to include '
+                          'them.',
+                          action='store_true')
+
+    optional.add_argument('--minThreshold',
+                          default=None,
+                          type=float,
+                          help='Numeric value. Any region containing a '
+                          'value that is equal or less than this numeric '
+                          'value will be skipped. This is useful to skip, '
+                          'for example, genes where the read count is zero '
+                          'for any of the bins. This could be the result of '
+                          'unmappable areas and can bias the overall results.')
+
+    optional.add_argument('--maxThreshold',
+                          default=None,
+                          type=float,
+                          help='Numeric value. Any region containing a value '
+                          'that is equal or higher that this numeric value '
+                          'will be skipped. The maxThreshold is useful to '
+                          'skip those few regions with very high read counts '
+                          '(e.g. major satellites) that may bias the average '
+                          'values.')
+
+    # in contrast to other tools,
+    # computeMatrix by default outputs
+    # messages and the --quiet flag supresses them
+    optional.add_argument('--quiet', '-q',
+                          help='Set to remove any warning or processing '
+                          'messages.',
+                          action='store_true')
+
+    optional.add_argument('--scale',
+                          help='If set, all values are multiplied by '
+                          'this number.',
+                          type=float,
+                          default=1)
+    optional.add_argument('--numberOfProcessors', '-p',
+                          help='Number of processors to use. Type "max/2" to '
+                          'use half the maximum number of processors or "max" '
+                          'to use all available processors.',
+                          metavar="INT",
+                          type=numberOfProcessors,
+                          default=cfg.config.get('general',
+                                                 'default_proc_number'),
+                          required=False)
+    return parser
+
+
+def process_args(args=None):
+    args = parse_arguments().parse_args(args)
+
+    if args.quiet is False:
+        args.verbose = True
+    else:
+        args.verbose = False
+
+    if args.command == 'scale-regions':
+        args.nanAfterEnd = False
+        args.referencePoint = None
+    elif args.command == 'reference-point':
+        if args.beforeRegionStartLength == 0 and \
+                args.afterRegionStartLength == 0:
+            sys.stderr.write("\nUpstrean and downstream regions are both "
```

Comment:
I removed a bunch of those cases as part of my PEP8 clean up. Taking about PEP8, I still have a lot of camelCase variables that are harder to change.